build/debs/kubelet.service

```diff
@@ -1,12 +0,0 @@
-[Unit]
-Description=kubelet: The Kubernetes Node Agent
-Documentation=http://kubernetes.io/docs/
-
-[Service]
-ExecStart=/usr/bin/kubelet
-Restart=always
-StartLimitInterval=0
-RestartSec=10
-
-[Install]
-WantedBy=multi-user.target
```

Comment:
I also think it's pretty terrible to not have a basic systemd unit for kubelet in k/k :/

Reply:
IMHO we can have the specs in k/k only if k/release is pulling them from here.
specs can be considered versioned source, but it must be at a single location.